Mathlib/Data/Prod/Basic.lean

```diff
@@ -60,21 +60,31 @@ theorem map_snd' (f : α → γ) (g : β → δ) : Prod.snd ∘ map f g = g ∘
 
 -- Porting note: `@[simp]` tag removed because auto-generated `mk.injEq` simplifies LHS
 -- @[simp]
-theorem mk.inj_iff {a₁ a₂ : α} {b₁ b₂ : β} : (a₁, b₁) = (a₂, b₂) ↔ a₁ = a₂ ∧ b₁ = b₂ :=
+theorem mk_inj {a₁ a₂ : α} {b₁ b₂ : β} : (a₁, b₁) = (a₂, b₂) ↔ a₁ = a₂ ∧ b₁ = b₂ :=
   Iff.of_eq (mk.injEq _ _ _ _)
 
-theorem mk.inj_left {α β : Type*} (a : α) : Function.Injective (Prod.mk a : β → α × β) := by
+@[deprecated (since := "2025-03-06")] alias mk.inj_iff := mk_inj
+
+theorem mk_left_injective {α β : Type*} (a : α) : Function.Injective (Prod.mk a : β → α × β) := by
```

Comment:
Actually: are you sure this lemma should be named "left"? This seems to contradict the naming convention (see below).